recipe/meta.yaml

```diff
@@ -1,21 +1,21 @@
 {% set name = "kwant" %}
-{% set version = "1.4.0a1" %}
+{% set version = "1.4.0rc0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://downloads.kwant-project.org/kwant/kwant-{{ version }}.tar.gz
-  sha256: e43e9ed928ff3e7cbe8e7b9e4b671871a36c72dd1000548746098af89f102a67
+  url: http://josephweston.org/uploads/kwant-{{ version }}.tar.gz
+  sha256: eb487b7d1f4f859255c5ec1901b029a792e1e03b918f1a169c57f612a48bc366
   patches:
     - skip_tests.patch  # [win]
 
 build:
   skip: true  # [py2k]
   skip: true  # [py==37]
```

Comment:
We shouldn't be skipping `py==37` anymore.

Reply:
this was added as part of some PR, I `git blame`d it but the commit message was just "disable Python 3.7" I left it as is for now, but was planning to re-enable once the rest of the builds pass

Reply:
It was because Sympy 1.1 wasn't build for Python 3.7.